src/components/CategoriesList.js

```diff
@@ -8,7 +8,7 @@ export const CategoriesList = (props) => {
   return (
     <div className="Categories-list"> 
       {
-        categories.map(category => <Category name={category.name} image={category.image}/>) 
+        categories.map(category => <Category key={category.name} name={category.name}/>) 
```

Comment:
Why do we need a key prop here? Looks like the name is being used for everything currently.